src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const longTermDiscount = 50;
+  const shortTermDiscount = 20;
+
+  if (days >= longTerm) {
+    result -= longTermDiscount;
+
+    return result;
```

Comment:
You could also write this for simplicity:
```suggestion
    return result - longTermDiscount;
```